snapcraft/file_utils.py

```diff
@@ -429,4 +430,17 @@ def _remove_readonly(func, path, excinfo):
 
 def rmtree(path: str) -> None:
     """Cross-platform rmtree implementation."""
-    shutil.rmtree(path, onerror=_remove_readonly)
\ No newline at end of file
+    shutil.rmtree(path, onerror=_remove_readonly)
+
+
+def format_linux_path(path: str) -> str:
+    """Convert native OS path to linux path.
+    Note that this is really only useful for relative paths...
+    e.g. "some\\path\\here" -> "some/path/here"
+    """
+
+    if sys.platform == "win32":
+        parts = pathlib.WindowsPath(path).parts
```

Comment:
why not pathlib.PureWindowsPath(...).to_posix()?

Reply:
done.